apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -102,6 +107,7 @@ const initialState: AichatState = {
   viewMode: ViewMode.EDIT,
   saveInProgress: false,
   currentSaveType: undefined,
+  userHasAichatAccess: undefined,
```

Comment:
Should we initialize this to `false` rather than `undefined`? Or do we think it might be beneficial to have `undefined` as an indeterminate state?

Reply:
Oh yeah - on an earlier iteration, wanted to distinguish between `undefined` and `false` but now this isn't needed since we're only sending currently implemented `aichat` analytics for authorized users. Will update!